mail/mutt/Portfile

```diff
@@ -3,9 +3,8 @@
 PortSystem          1.0
 
 name                mutt
-version             2.2.10
+version             2.2.12
 categories          mail
-platforms           darwin
 license             GPL-2+
 maintainers         {l2dy @l2dy}
```

Comment:
@l2dy: any reason why this can't be `openmaintainer`?

Reply:
I prefer keeping it so for the time being. Non-`openmaintainer` ports do not limit contributions. It just gives maintainers a 72-hour window to respond to changes, and reject invalid PRs like #20696.